activities/standard-maintenance/index.md

```diff
@@ -6,7 +6,7 @@ type: index
 
 The Foundation for Public Code and its codebase stewards maintain the [Standard for Public Code](https://standard.publiccode.net).
```

Comment:
This gives an incomplete view of our Standard maintenance (it's not just community calls.) 
Suggestion: add that we also update the Standard according to CONTRIBUTING and GOVERNANCE. See the Changelog for examples of past Standard maintenance.

Reply:
That part is not new, and fixing it shouldn't be a blocker for fixing #701. I was already hesitant in adding the lost words in this PR due to scope creep.

Reply:
I created #776 to address the incompleteness.